browsers-ng.js

```diff
@@ -1,60 +1,167 @@
 /* eslint-disable */
-'use strict';
+// 'use strict';
```

Comment:
Can we just remove this line instead of commenting it out?

Reply:
I don't even remember if I did this